package.json

```diff
@@ -83,25 +79,9 @@
     "scripts/spine/playcanvas-spine.3.8.mjs"
   ],
   "files": [
-    "build/playcanvas.js",
-    "build/playcanvas.min.js",
-    "build/playcanvas.dbg.js",
-    "build/playcanvas.prf.js",
-    "build/playcanvas.mjs",
-    "build/playcanvas.min.mjs",
-    "build/playcanvas.dbg.mjs",
-    "build/playcanvas.prf.mjs",
-    "build/playcanvas.d.ts",
-    "build/playcanvas-extras.js",
-    "build/playcanvas-extras.mjs",
-    "build/playcanvas-extras/index.js",
+    "build/playcanvas*",
     "scripts",
-    "LICENSE",
-    "package.json",
-    "README.md",
-    "README-ja.md",
-    "README-kr.md",
-    "README-zh.md"
+    "README*.md"
```

Comment:
So presumably the `LICENSE` and `package.json` are implicitly included and don't need to be explicitly listed here?